packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -114,8 +114,25 @@ export class FungibleTokenTransferController implements ReactiveController {
     void this.filterDestinationNetworksAndResources(network);
   };
 
+  getSenderDefaultDestinationAddress = (): void => {
+    if (!this.sourceNetwork || !this.destinationNetwork) {
+      this.destinationAddress = '';
+      return;
+    }
+
+    const isSameNetwork =
+      this.sourceNetwork.chainId === this.destinationNetwork.chainId;
+    const isSameType = this.sourceNetwork.type === this.destinationNetwork.type;
+
+    this.destinationAddress =
+      isSameNetwork || isSameType
+        ? this.walletContext.value?.evmWallet?.address || ''
+        : '';
+  };
+
   onDestinationNetworkSelected = (network: Domain | undefined): void => {
     this.destinationNetwork = network;
+    this.getSenderDefaultDestinationAddress();
```

Comment:
```suggestion
    this.maybeSetSenderDefaultDestinationAddress();
```

Reply:
I'm not sure about "maybe" part, but as it's not returning the value the "setSenderDefaultDestinationAddress" looks sufficient for me. As empty value is also a value 😅

Reply:
I've added maybe because we aren't always setting it haha